.vscode/settings.json

```diff
@@ -3,10 +3,21 @@
         "editor.defaultFormatter": "charliermarsh.ruff",
         "editor.formatOnSave": true,
         "editor.codeActionsOnSave": {
-            "source.fixAll": true,
-            "source.organizeImports": true
+            "source.fixAll": "explicit",
+            "source.organizeImports": "explicit"
```

Comment:
esto es para que las moddicaciones que no son formatting (eliminar imports, cambios como `for i in dic.keys()` -> `for i in dic`)no se haga de forma automática, o sólo para que ciertos cambios no se hagan de forma automática?